tests/test_autocommit.py

```diff
@@ -0,0 +1,11 @@
+import sys, os
+
+def test():
+    "Verify autocommit just before program exits."
+    assert(0 == os.system('PYTHONPATH=. %s tests/autocommit.py' % sys.executable))
+    # The above script relies on the autocommit feature working correctly.
+    # Now, let's check if it actually worked.
+    import sqlitedict
```

Comment:
Is there a reason for this import to be nested inside the function? If not, please put it at module level.

Reply:
There is no reason.  I will move it to the module level.